docs/source/serialization.rst

```diff
@@ -48,32 +48,64 @@ will be serialized to
       "encoding": "utf-8"
     }
 
-File Fields
------------
+File / Image Fields
+-------------------
 
 Download (serialization)
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-For download, the file field will simply be serialized to a string that contains the download URL for the file.
+For download, a file field will be serialized to a mapping that contains the
+file's most basic metadata, and a hyperlink that the client can follow to
+download the file:
 
 .. code:: json
 
       {
         "...": "",
         "@type": "File",
         "title": "My file",
-        "file": "http://localhost:55001/plone/file/@@download/file"
+        "file": {
+          "content-type": "application/pdf",
+          "download": "http://localhost:55001/plone/file/@@download/file",
+          "filename": "file.pdf",
+          "size": 74429
+        }
       }
 
-That URL points to the regular Plone download view.
+That URL in the ``download`` property points to the regular Plone download
+view. The client can send a ``GET`` request to that URL with an ``Accept``
+header containing the MIME type indicated in the ``content-type`` property,
+and will get a response containing the file.
+
+Image fields are serialized in the same way, except that their serialization
+contains their ``width`` and ``height``, and an additional property
+``scales`` that contains a mapping with the available image scales:
+
+.. code:: json
+
+    {
+      "icon": {
+        "download": "http://localhost:55001/plone/image/@@images/image/icon",
+        "height": 32,
+        "width": 24
+      },
+      "large": {
+        "download": "http://localhost:55001/plone/image/@@images/image/large",
+        "height": 768,
+        "width": 576
+      },
+      "..." : {}
+    }
+
+(For non-square images, those will be the actual scaled images' dimensions,
```

Comment:
I would simply drop this remark because maintaining aspect ratio is not related to square or non-square images. One could also define non-square image sizes (e.g. 600 x 400).